cypress/jenkins/init.sh

```diff
@@ -92,13 +93,33 @@ create_initial_clusters() {
     TARFILE="helm-v${HELM_VERSION}-linux-amd64.tar.gz"
     curl -L -o "${TARFILE}" "https://get.helm.sh/${TARFILE}"
     tar -C "${WORKSPACE}/bin" --strip-components=1 -xzf "${TARFILE}"
-    helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
-    helm repo update
+    if [[ -n "${RANCHER_HELM_REPO}" ]]; then
+      if [[ "${RANCHER_HELM_REPO}" == "prime" ]]; then
+        RANCHER_CHART_URL=https://charts.rancher.com/server-charts/prime
+        helm repo add rancher-prime "${RANCHER_CHART_URL}"
+        helm repo update
+      elif [[ "${RANCHER_HELM_REPO}" == "alpha" ]]; then
+        RANCHER_CHART_URL=https://releases.rancher.com/server-charts/alpha
+        helm repo add rancher-alpha "${RANCHER_CHART_URL}"
+        helm repo update
+      elif [[ "${RANCHER_HELM_REPO}" == "stable" ]]; then
+        RANCHER_CHART_URL=https://releases.rancher.com/server-charts/stable
+        helm repo add rancher-stable "${RANCHER_CHART_URL}"
+        helm repo update
+      else
+        RANCHER_CHART_URL=https://releases.rancher.com/server-charts/latest
+        helm repo add rancher-latest "${RANCHER_CHART_URL}"
+        helm repo update
+      fi
+      corral config vars set rancher_chart_repo "${RANCHER_HELM_REPO}"
+      url_string=$(echo "${RANCHER_CHART_URL}" | grep -o '.*server-charts')
+      corral config vars set rancher_chart_url "${url_string}"
+    fi
     version_string=$(echo "${RANCHER_IMAGE_TAG}" | cut -f1 -d"-")
     if [[ "${RANCHER_IMAGE_TAG}" == "head" ]]; then
-        RANCHER_VERSION=$(helm search repo rancher-latest --devel --versions | sed -n '1!p' | head -1 | cut -f2 | tr -d '[:space:]')
+      RANCHER_VERSION=$(helm search repo "${RANCHER_HELM_REPO}" --devel --versions | sed -n '1!p' | head -1 | cut -f2 | tr -d '[:space:]')
```

Comment:
It looks like you're calling the helm repo `"rancher-${RANCHER_HELM_REPO}"` so this probably won't return anything currently

Reply:
ah nice catch. oddly enough the same value is returned with or w/o the 'rancher-' prefix but I'll make the change since that is the correct naming

Reply:
Ohhh interesting! I just checked locally where I have a lot more helm repos, and it returns multiple charts when I just do `latest` vs. only the specified chart when I do `rancher-latest`.. TIL!

```
$ helm search repo latest --devel --versions
NAME                        	CHART VERSION   	APP VERSION      	DESCRIPTION                                       
rancher-latest/rancher      	2.9.1           	v2.9.1           	Install Rancher Server to manage Kubernetes clu...
rancher-latest/rancher      	2.9.1-rc6       	v2.9.1-rc6       	Install Rancher Server to manage Kubernetes clu...
...
rancher-latest/rancher      	2.0.4           	v2.0.4           	Install Rancher Server to manage Kubernetes clu...
bitnami/wildfly             	21.0.4          	33.0.1           	Wildfly is a lightweight, open source applicati...
bitnami/wildfly             	21.0.3          	33.0.0           	Wildfly is a lightweight, open source applicati...
...
bitnami/wildfly             	13.3.6          	26.1.1           	Wildfly is a lightweight, open source applicati...
rancher-charts/rancher-istio	104.4.0+up1.22.1	1.22.1           	A basic Istio setup that installs with the isti...
rancher-charts/rancher-istio	104.3.0+up1.21.1	1.21.1           	A basic Istio setup that installs with the isti...
...
rancher-charts/rancher-istio	101.0.0+up1.14.3	1.14.3           	A basic Istio setup that installs with the isti...
```
vs.
```
$ helm search repo rancher-latest --devel --versions
NAME                  	CHART VERSION   	APP VERSION      	DESCRIPTION                                       
rancher-latest/rancher	2.9.1           	v2.9.1           	Install Rancher Server to manage Kubernetes clu...
rancher-latest/rancher	2.9.1-rc6       	v2.9.1-rc6       	Install Rancher Server to manage Kubernetes clu...
...

rancher-latest/rancher	2.0.4           	v2.0.4           	Install Rancher Server to manage Kubernetes clu...
```